frontend/src/components/Game.vue

```diff
@@ -1,36 +1,44 @@
 <template>
     <div id="game">
-        <div v-if="response.status != 'Waiting for an opponent'" id="board">
-            <div id="my-score">0</div>
-            <div id="opponent-score">0</div>
-            <div v-for="(row, rownum) in response.rows"
-                :key="row.$index"
-                class="game-row"
-                :rownum="rownum"
-                v-on:dragover="allowDrop"
-                v-on:drop="dropOnRow">
-                <div v-for="card in row"
-                    :key="card.$index"
+        <div v-if="response.status != 'Waiting for an opponent'"
+            id="board"
+            class="row">
+            <div id="my-score" class="col text-right">0</div>
+            <div id="opponent-score" class="col text-left">0</div>
```

Comment:
We still haven't defined what "my score" and "opponent score" means. Number of rows won at the moment? It's necessary to specify that for whoever implements it.

Reply:
Yes, and that is already another issue, this is just about styles